### ISSUE TITLE
Ensembl release

### ISSUE THREAD

**Issue by aiqc** [ewels/AWS-iGenomes#19]
Is there a way to tell which release of Ensembl (e.g. 109) files were created with? Should that be a dropdown in the sync command builder?
https://ewels.github.io/AWS-iGenomes/

**Comment by aiqc** [comment]
Also, which assembly were these created with?

**Comment by ewels** [comment]
Yes, the AWS-iGenomes references are now getting very old. See https://nf-co.re/docs/usage/reference_genomes :

> [!WARNING]
> The transcriptome and GTF files in iGenomes are vastly out of date with respect to current annotations from Ensembl e.g. human iGenomes annotations are from Ensembl release 75, while the current Ensembl release is 108. Please consider downloading and using a more updated version of your reference genome as outlined in the next section.

Most of the assets in AWS-iGenomes are taken from the [illumina iGenomes project](https://support.illumina.com/sequencing/sequencing_software/igenome.html) (as mentioned in the readme). This project has not been updated my illumina in many years, and AWS-iGenomes has grown stale with it. Currently, AWS-iGenomes updates are extremely manual and rely on a sole maintainer (me).

We are (and have been for some time) actively looking into a replacement for AWS-iGenomes back end that will be much easier to maintain. Future efforts will instead be community driven and highly automated.
